src/raw/response.rs

```diff
@@ -309,29 +309,101 @@ response_type! {
 /// Type of the [`UserListensResponse::payload`] field.
 #[derive(Debug, Deserialize)]
 pub struct UserListensPayload {
+    /// The count of listen retrived from the database. 
```

Comment:
I'd say just leave the field docs out for now, only adding them to some of the types would be inconsistent.